<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVMTrainner</class>
 <widget class="QMainWindow" name="SVMTrainner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVMTrainner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QTabWidget" name="mainTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <widget class="QWidget" name="plateTab">
       <attribute name="title">
        <string>车牌训练</string>
       </attribute>
       <widget class="QPushButton" name="plateTrainButton">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>0</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>进行训练</string>
        </property>
       </widget>
       <widget class="QPushButton" name="platePropertyButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>训练配置</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadPlateSetButton">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>0</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>装入训练库</string>
        </property>
       </widget>
       <widget class="QPushButton" name="savePlateTrainButton">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>保存训练成果</string>
        </property>
       </widget>
       <widget class="QTreeWidget" name="plateSampleTree">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>141</width>
          <height>541</height>
         </rect>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <attribute name="headerMinimumSectionSize">
         <number>50</number>
        </attribute>
        <column>
         <property name="text">
          <string>样本集</string>
         </property>
        </column>
       </widget>
       <widget class="QTreeWidget" name="plateTestTree">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>40</y>
          <width>141</width>
          <height>541</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>测试集</string>
         </property>
        </column>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>40</y>
          <width>331</width>
          <height>541</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="plateGreyBox">
           <property name="title">
            <string>灰度图</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="plateOriginalLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="plateHogBox">
           <property name="title">
            <string>Hog</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="plateHogLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="plateSampleImfoBox">
           <property name="title">
            <string>Imfo</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QWidget" name="plateSampleImfo" native="true">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <layout class="QGridLayout" name="plateSampleImfoGrid">
                   <item row="0" column="1" colspan="2">
                    <widget class="QLabel" name="plateSampleName">
                     <property name="font">
                      <font>
                       <family>Times New Roman</family>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="plateHogSizeLabel">
                     <property name="font">
                      <font>
                       <family>Times New Roman</family>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>HOG维度：</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLabel" name="plateHogSize">
                     <property name="font">
                      <font>
                       <family>Times New Roman</family>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="2">
                    <widget class="QPushButton" name="plateSampleChangeButton">
                     <property name="text">
                      <string>更改标签</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="1">
                    <widget class="QComboBox" name="plateSampleLabelCombo">
                     <property name="font">
                      <font>
                       <family>Times New Roman</family>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <item>
                      <property name="text">
                       <string>非车牌</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>普通车牌</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="3" column="0">
                    <widget class="QLabel" name="plateSampleLabelLabel">
                     <property name="font">
                      <font>
                       <family>Times New Roman</family>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>标签：</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QLabel" name="plateSampleNameLabel">
                     <property name="font">
                      <font>
                       <family>Times New Roman</family>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>文件名：</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLabel" name="plateSampleSize">
                     <property name="font">
                      <font>
                       <family>Times New Roman</family>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="plateSampleSizeLabel">
                     <property name="font">
                      <font>
                       <family>Times New Roman</family>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>原图尺寸：</string>
                     </property>
                    </widget>
                   </item>
                   <item row="4" column="2">
                    <widget class="QPushButton" name="plateSampleDeleteButton">
                     <property name="text">
                      <string>删除样本</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="plateTestImfo" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <layout class="QGridLayout" name="gridLayout">
                     <item row="2" column="1">
                      <widget class="QLabel" name="testPlateHogSize">
                       <property name="font">
                        <font>
                         <family>Times New Roman</family>
                         <pointsize>12</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QLabel" name="testPlateSampleSize">
                       <property name="font">
                        <font>
                         <family>Times New Roman</family>
                         <pointsize>12</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="testPlateSampleSizeLabel">
                       <property name="font">
                        <font>
                         <family>Times New Roman</family>
                         <pointsize>12</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string>原图尺寸：</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <widget class="QLabel" name="testPlateHogSizeLabel">
                       <property name="font">
                        <font>
                         <family>Times New Roman</family>
                         <pointsize>12</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string>HOG维度：</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="0">
                      <widget class="QLabel" name="testPlateSampleNameLabel">
                       <property name="font">
                        <font>
                         <family>Times New Roman</family>
                         <pointsize>12</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string>文件名：</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1" colspan="2">
                      <widget class="QLabel" name="testPlateSampleName">
                       <property name="font">
                        <font>
                         <family>Times New Roman</family>
                         <pointsize>10</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QWidget" name="rightPlateTag" native="true">
                     <layout class="QHBoxLayout" name="horizontalLayout_7">
                      <item>
                       <widget class="QLabel" name="testPlateSampleLabelLabel">
                        <property name="font">
                         <font>
                          <family>Times New Roman</family>
                          <pointsize>10</pointsize>
                         </font>
                        </property>
                        <property name="text">
                         <string>正确标签：</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QComboBox" name="plateTestRighTagCombo">
                        <property name="font">
                         <font>
                          <family>Times New Roman</family>
                          <pointsize>9</pointsize>
                         </font>
                        </property>
                        <item>
                         <property name="text">
                          <string>未知</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>非车牌</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>普通车牌</string>
                         </property>
                        </item>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="testPlateChangeTagButton">
                        <property name="font">
                         <font>
                          <family>Times New Roman</family>
                          <pointsize>8</pointsize>
                         </font>
                        </property>
                        <property name="text">
                         <string>更改标签</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QWidget" name="plateTestResult" native="true">
                     <layout class="QHBoxLayout" name="horizontalLayout_2">
                      <item>
                       <widget class="QLabel" name="plateTestResultLabel">
                        <property name="font">
                         <font>
                          <family>Times New Roman</family>
                          <pointsize>10</pointsize>
                         </font>
                        </property>
                        <property name="text">
                         <string>测试结果：</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QComboBox" name="plateTestResultCombo">
                        <property name="font">
                         <font>
                          <family>Times New Roman</family>
                          <pointsize>9</pointsize>
                         </font>
                        </property>
                        <item>
                         <property name="text">
                          <string>非车牌</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>普通车牌</string>
                         </property>
                        </item>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="correctPlateButton">
                        <property name="font">
                         <font>
                          <family>Times New Roman</family>
                          <pointsize>8</pointsize>
                         </font>
                        </property>
                        <property name="text">
                         <string>加入训练库</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="loadPlateTestSetButton">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>0</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>载入测试集</string>
        </property>
       </widget>
       <widget class="QPushButton" name="generatePlateTestSetButton">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>0</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>生成测试集</string>
        </property>
       </widget>
       <widget class="QPushButton" name="startPlateTestButton">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>0</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>进行测试</string>
        </property>
       </widget>
       <widget class="QPushButton" name="singlePlateTestButton">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>0</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>单张图测试</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadPlateTrainResultButton">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>0</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>加载训练结果</string>
        </property>
       </widget>
       <widget class="QPushButton" name="plateRefreshButton">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>0</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
       <widget class="QPushButton" name="equalizePlateTrainButton">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>0</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>均衡训练集</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="charTab">
       <attribute name="title">
        <string>字符训练</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
